static/chromium/devtools/main.go

```diff
@@ -0,0 +1,23 @@
+package main
```

Comment:
It makes no sense to copy this binary source code if there are no changes. Instead I would copy `static/chrome/devtools` before building Docker image.

Reply:
Agreed to skip DevTools atm due to an issue accessing Chrome sub-folders within Chromium docker context